openproject/client.py

```diff
@@ -86,8 +87,11 @@ def _api_payload_from_kwargs(self, **kwargs: WorkPackage):
         data = {api_args: kwargs[args] for args, api_args in items if args in kwargs}
         return data
 
-    def list(self):
-        return self.client._send_request("GET", "work_packages")
+    def list(self, **kwargs):
+        project_value = kwargs["project"]
+        filters = [{"project": {"operator": "=", "values": project_value}}]
+        params = {"filters": json.dumps(filters)}
+        return self.client._send_request("GET", "work_packages", params=params)
```

Comment:
Let's use typed arguments to make the developer experience better here and also take into account the case where someone does not necessarily want to provide any filters.

**Example**
```py
def list(self, project: int|None):
    params = {}
    if project:
        filters = [{"project": {"operator": "=", "values": str(project)}}]
        params["filters"] = json.dumps(filters)
    return self.client._send_request("GET", "work_packages", params=params)
```